packages/scandipwa/src/component/ExpandableContentShowMore/ExpandableContentShowMore.style.scss

```diff
@@ -26,7 +26,7 @@
     }
 
     &-ExpandableChildren {
-        transition: .5s;
+        //transition: .5s;
```

Comment:
Please don't leave the commented-out code.
If you feel it's not necessary, remove it completely.

Reply:
BTW, how this is related to the menu item hover state?

Reply:
This is removed in the #4839